sdk/keyvault/keyvault-certificates/package.json

```diff
@@ -26,7 +26,7 @@
     "node": ">=8.0.0"
   },
   "files": [
-    "types/",
+    "types/keyvault-certificates.d.ts",
```

Comment:
Not publishing the down leveled types?

Reply:
I believe this was the feedback by @xirzec , am I wrong here?

(good news for me is that reverting this doesn't require re-recording things)

Reply:
AFAIK nobody can use the non-rolled up types since types for this since we declare the types to be the rollup file in the package.json and it doesn't import from the other files.

So including them doesn't actually do anything except make the package bigger.

Reply:
Oh and we don't seem to be using downlevel-dts in this package, so there's no downleveled rollup file.

Reply:
Okay, once you add downlevel-ing you can have something like this
![image](https://user-images.githubusercontent.com/10452642/85798823-0b3bd480-b6f3-11ea-85eb-c780d54c4014.png)

For now, the current change is good.